src/components/Announce/blogpost.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
Note: you might want to also check the merged_at property of the PR to ensure the PR was actually merged.  I think someone could randomly open a PR, put whatever they want in the text, it gets closed, and they can still tag it with release.